lib/Driver/Compilation.cpp

```diff
@@ -674,12 +688,27 @@ namespace driver {
     PerformJobsState(Compilation &Comp, std::unique_ptr<TaskQueue> &&TaskQueue)
       : Comp(Comp), ActualIncrementalTracer(Comp.getStatsReporter()),
         TQ(std::move(TaskQueue)) {
-      if (Comp.getShowIncrementalBuildDecisions() || Comp.getStatsReporter())
+      if (Comp.getEnableExperimentalDependencies())
+        ExpDepGraph.emplace(experimental_dependencies::DriverGraph());
```

Comment:
Nitpick: `emplace` takes constructor arguments for the wrapped type, so this should either be `ExpDepGraph.emplace()` or `ExpDepGraph = …`.

Reply:
Thanks.